packages/azure/attester/celo-attestations-run

```diff
@@ -21,7 +21,7 @@ TWILIO_AUTH_TOKEN=$(get_secret $ATTESTER_KEY_VAULT 'attesterTwilioAuthToken')
   --name $1 \
   --net=host \
   --entrypoint /bin/bash \
-  -v $DATA_DIR:$DATA_DIR \
+  -v $DATA_DIR:$DATA_DAFIR \
```

Comment:
maybe a typo? or is there a reason for this?

Reply:
It is indeed a typo.. good catch!